packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/liveness.test.ts

```diff
@@ -58,23 +58,25 @@ describe('Liveness Helper', () => {
       expect(faceMatchPercentage).toBe(0);
     });
 
+    // Note: if this test ever fails just add a console log for face and do a check and copy the value here
```

Comment:
Can you add a note about _why_ it might fail?

Reply:
oh yes it would only fail because we update the face detection, will add